src/frontend/components/InstructorView/index.tsx

```diff
@@ -65,12 +65,16 @@ interface InstructorViewProps {
 export const InstructorView = ({ children, resource }: InstructorViewProps) => {
   const canAccessDashboard =
     getDecodedJwt().permissions.can_update &&
-    false === getDecodedJwt().maintenance;
-  const message = canAccessDashboard
-    ? messages.title
-    : getDecodedJwt().maintenance
-    ? messages.maintenance
-    : messages.disabledDashboard;
```

Comment:
I'm not a fan of ternary nesting but in some case it improves readability that's why the rule `no-nested-ternary` is set to `warn`. So keep the code with nested ternaries should not be a lint error;